Anotasi_Image/reviewer/urls.py

```
@@ -0,0 +1,6 @@
+from django.urls import path
```

Ruff (0.8.2)

`django.urls.path` imported but unused

Remove unused import: `django.urls.path`

(F401)